components/automate-gateway/gateway/services.go

```diff
@@ -73,7 +73,8 @@ func (s *Server) RegisterGRPCServices(grpcServer *grpc.Server) error {
 	if err != nil {
 		return errors.Wrap(err, "create client for deployment service")
 	}
-	pb_deployment.RegisterDeploymentServer(grpcServer, handler.NewDeploymentServer(deploymentClient))
```

Comment:
now this is a bit hidden in your "Remove deprecated v1 IntrospectAllProjects" commit, isn't it? https://github.com/chef/automate/pull/480/commits/f4d4c74dff09c6d518c11e5b9028327afcadb255

I know it'll eventually get squashed, but still 😉

Reply:
Also I'd opt for either _changing the linter settings to flag this_ or _not caring_ 🙈 -- fixing stuff like this that isn't detected by the linter is an uphill battle, I suppose 😉

Reply:
It is an uphill battle, yes, but I really find it cumbersome to work with long lines. 🤷‍♂